source/navigation/routes.tsx

```diff
@@ -277,6 +272,11 @@ const SettingsStackScreens = () => (
 		<SettingsStack.Screen component={settings.PrivacyView} name="Privacy" />
 		<SettingsStack.Screen component={settings.LegalView} name="Legal" />
 		<SettingsStack.Screen component={settings.APITestView} name="APITest" />
+		<Stack.Screen
```

Comment:
Stack? Or SettingsStack?

Reply:
Oh. I missed changing which screen type it is wrapped with. Fun that even appears to work correctly in this setup.